src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-summary-page.mdx

```diff
@@ -90,4 +90,19 @@ For your OpenTelemetry data to appear in the **Summary** page, make sure it has
   </tbody>
 </table>
 
+### Distributed tracing insights [#distributed-tracing-insights]
+
+<img
```

Comment:
This is probably causing your build issues! Looks like this is being called, but was never imported into our images folder. If you add it in, the build should work.

Reply:
fixed